scripts/cat/cats.py

```diff
@@ -3121,13 +3111,13 @@ def set_faded(self):
         self.faded = True
 
         # Silhouette sprite
-        if self.age == "newborn":
+        if self.age == CatAgeEnum.NEWBORN:
             file_name = "faded_newborn"
-        elif self.age == "kitten":
+        elif self.age == CatAgeEnum.KITTEN:
             file_name = "faded_kitten"
-        elif self.age in ["adult", "young adult", "senior adult"]:
+        elif self.age in [CatAgeEnum.ADULT, CatAgeEnum.YOUNG_ADULT, CatAgeEnum.SENIOR_ADULT]:
```

Comment:
Consider adding a helper function for this in CatAgeEnum.

Reply:
Well, it's only used in one place...